KifuSF/DeliveryModularTableViewController.swift

```diff
@@ -74,49 +89,24 @@ class KFCDelivery: KFCModularTableView {
         }
     }
     
-    override func loadView() {
-        super.loadView()
-        
-        view.addSubview(actionButton)
-        configureLayoutConstraints()
-        actionButton.addTarget(
-            self,
-            action: #selector(pressActionButton(_:)),
-            for: .touchUpInside
-        )
-    }
-    
-    override func viewDidLoad() {
-        super.viewDidLoad()
-        
-        configureStyling()
-    }
-    
-    override func viewDidAppear(_ animated: Bool) {
-        super.viewDidAppear(animated)
-        
-        modularTableView.contentInset.bottom = actionButton.frame.height + 16
-        modularTableView.scrollIndicatorInsets.bottom = actionButton.frame.height + 16
-    }
-    
-    override func retrieveProgressItem() -> KFPModularTableViewItem? {
+    override func retrieveProgressItem() -> ModularTableViewItem? {
         guard let deliveryStep = self.delivery?.status.step else {
             return nil
         }
         
         return KFMProgress(currentStep: deliveryStep, ofType: .delivery)
     }
     
-    override func retrieveInProgressDonationDescription() -> KFPModularTableViewItem? {
+    override func retrieveInProgressDonationDescription() -> ModularTableViewItem? {
         guard let deliveryDescription = self.delivery?.inProgressDescriptionForVolunteer else {
             return nil
         }
         
         return deliveryDescription
     }
     
-    override func retrieveEntityInfoItem() -> KFPModularTableViewItem? {
-        guard let delivery = self.delivery else {
+    override func retrieveEntityInfoItem() -> ModularTableViewItem? {
+        guard let _ = self.delivery else {
```

Comment:
since this is part of a `TODO`, is there a reason to remove this warning of *unused variable*?